src/main/java/org/jabref/gui/importer/ImportEntriesDialog.java

```diff
@@ -76,14 +76,14 @@ public ImportEntriesDialog(BibDatabaseContext database, BackgroundTask<ParserRes
         Button btn = (Button) this.getDialogPane().lookupButton(importButton);
         btn.disableProperty().bind(booleanBind);
 
-        setResultConverter(button -> {
+       setResultConverter(button -> {
```

Comment:
Hi @jcampbell11245, thanks for your contribution.
Tiny issue: Indentation is moved to the left. ;-)